src/test/java/org/fcrepo/client/integration/FcrepoClientIT.java

```diff
@@ -342,8 +377,6 @@ public void testGetRange() throws Exception {
         final String mimetype = "text/plain";
         final String bodyContent = "Hello world";
         final FcrepoResponse response = client.post(new URI(serverAddress))
-                .digestMd5("f0ef7081e1539ac00ef5b761b4fb01b3")
```

Comment:
I removed them since they seemed unrelated to testing the Range header on a GET request, and since there are now more methods directly testing digests on POST requests in this IT.  There wasn't any functional reason to remove them, so I would be happy to restore them if you'd like